app/views/locations/index.html.erb

```diff
@@ -1,5 +1,11 @@
+<%= form_tag(locations_path, :method => "get", id: "search-form") do %>
+<%= text_field_tag :search, params[:search], placeholder: "Search Locations" %>
+<%= submit_tag "Search" %>
```

Comment:
This line and the one above it should be indented, so as to be explicitly nested under the `form_tag` (and to keep the `end` from looking utterly out of place.